test/cpp/qps/qps_diff.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python2.7
```

Comment:
Should this file be in `test/cpp`? It looks like it might fit in better in `tools/profiling`.